forester/docker-compose.yml

```diff
@@ -22,7 +20,71 @@ services:
       - PGADMIN_DEFAULT_PASSWORD=photon
```

Comment:
should we make this an env variable?
(Same question for postgres login above.)

Reply:
yes we can, and probably should, it's just a default test value, it implies that the end user should replace it with something more secure